Services/CO.CDP.EntityVerification.Tests/Ppon/OrganisationRegisteredEventHandlerTests.cs

```diff
@@ -13,20 +13,17 @@ public async Task Action_GeneratesPponIdAndPersistsAndPublishes()
     {
         var pponRepository = new Mock<IPponRepository>();
         var pponService = new Mock<IPponService>();
-        var publisher = new Mock<IPublisher>();
         var generatedPpon = "92be415e5985421087bc8fee8c97d338";
 
         pponService.Setup(x => x.GeneratePponId()).Returns(generatedPpon);
 
-        var handler = new OrganisationRegisteredEventHandler(pponService.Object, pponRepository.Object, publisher.Object);
+        var handler = new OrganisationRegisteredEventHandler(pponService.Object, pponRepository.Object);
         var @event = GivenOrganisationRegisteredEvent();
 
         await handler.Handle(@event);
 
         pponRepository.Verify(
             s => s.Save(It.Is<EntityVerification.Persistence.Ppon>(p => p.PponId == generatedPpon)), Times.Once);
-        publisher.Verify(
-              s => s.Publish(It.Is<EntityVerification.Persistence.Ppon>(p => p.PponId == generatedPpon)), Times.Once);
```

Comment:
Why are we removing this?

Reply:
Because that's part of DP-385

Reply:
Sure, but we shouldn't be removing previously added features :)